src/onegov/org/forms/directory.py

```diff
@@ -334,9 +375,24 @@ class DirectoryBaseForm(Form):
             ('submitter_address', _("Address")),
             ('submitter_phone', _("Phone")),
         ),
-        fieldset=_("Submitter")
+        fieldset=_("Submitter"),
+        depends_on=('is_faq', 'n'),
     )
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
I don't like the ctor approach as users can not adapt the `structure` of their faq directories as the ctor will overwrite the values upon submit.

Reply:
You could set `field.default` in `on_request` instead. I would avoid overwriting `__init__` since we have plenty of other ways to hook control flow.